### PR TITLE
Add graphs for the queue counts of pfqueue

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Add graphs for the queue counts of pfqueue

# Impacts
pfmon jobs (adds a new one)
pfqueue status page

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Added graph to track pfqueue queues count in the queue status page.

### CODE REVIEW

**Comment by julsemaan** [comment]
I've put it to 7.0 given its small code changes and very useful IMO

We could postpone it to 7.1 without any problem though

**Comment by dwlfrth** [comment]
Will review and merge

File: lib/pf/pfmon/task/queue_stats.pm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    my $queue_counts = pf::pfqueue::stats->new->queue_counts;
+    for my $info (@{$queue_counts}) {
+        my $queue_name = $info->{name};
+        my $count = $info->{count};
+        $logger->debug("Setting queue count of $queue_name to $count in statsd");
+        $statsd->gauge("pfqueue.stats.queue_counts.$queue_name", $count, 1);
+    }
+}
+
```
Comment (jrouzierinverse) [inline lib/pf/pfmon/task/queue_stats.pm:44]: If you use ```pf::pfqueue::stats->stats_data```

We can get an even bigger picture of the state of the queue

```perl
my $stats_data = pf::pfqueue::stats->new->stats_data;
```
```$stats_data``` provided the following information

```perl
 {
          'queue_counts' => [
                              {
                                'count' => 0,
                                'name' => 'general'
                              },
                              {
                                'count' => 0,
                                'name' => 'pfdhcplistener'
                              },
                              {
                                'count' => 0,
                                'name' => 'pfdetect'
                              },
                              {
                                'count' => 0,
                                'name' => 'pfsnmp'
                              }
                            ],
          'miss_counters' => [
                               {
                                 'count' => '12194',
                                 'name' => 'api:echo',
                                 'queue' => 'general'
                               },
                               {
                                 'count' => '12194',
                                 'name' => 'api:sleep',
                                 'queue' => 'general'
                               },
                               {
                                 'count' => '12196',
                                 'name' => 'api:throw',
                                 'queue' => 'general'
                               }
                             ],
          'counters' => [
                          {
                            'count' => '0',
                            'name' => 'api:echo',
                            'queue' => 'general'
                          },
                          {
                            'count' => '0',
                            'name' => 'api:sleep',
                            'queue' => 'general'
                          },
                          {
                            'count' => '0',
                            'name' => 'api:throw',
                            'queue' => 'general'
                          }
                        ]
 };
```

Comment (julsemaan) [reply lib/pf/pfmon/task/queue_stats.pm:44]: Well for now, I've just made it to track the queue counts simply as its only used for one graph now.

I don't like having too much whispr DBs that we're now using in the admin graphs given they take their fair share of disk space

**Comment by dwlfrth** [comment]
LGTM

**Pull request merged**